sw/device/silicon_creator/manuf/lib/provisioning_functest.c

```diff
@@ -22,6 +23,11 @@
 
 OTTF_DEFINE_TEST_CONFIG(.enable_uart_flow_control = true);
 
+static_assert(
+    (sizeof(wrapped_rma_unlock_token_t) % sizeof(uint32_t)) == 0,
+    "Size of the wrapped_rma_unlock_token_t struct must the an even multiple "
+    "of 32-bit words since it is written to the non-volatile flash region.");
```

Comment:
Post-merge comment: I think this should be 64 bits rather than 32 bits, as the flash word size is 64 bits:
https://opentitan.org/book/hw/ip/flash_ctrl/doc/theory_of_operation.html#flash-default-configuration

Reply:
fixed in #18672